src/blocks/Slider/Slider.tsx

```diff
@@ -276,19 +277,26 @@ export const SliderBlock = (props: WithChildren<SliderProps>) => {
         } else {
             currentSlideNumber = index + 1;
         }
+        return currentSlideNumber;
+    };
+    const isVisibleSlide = (index: number) => {
+        const currentIndexDiff = index - currentIndex;
 
+        return (
+            slidesCountByBreakpoint > 0 &&
```

Comment:
Something wrong with a slider. I can tab trough cards and then if I click to the arrow it doesn't work right
https://disk.yandex.ru/i/7dRgmxUNMis7iA

Reply:
Yep, this is an issue I think came from react-slick, I didn't manage to find a solution better than wrap each slide with `inert` attribute. But it may be risky, as slides not visible for the user, will be unavailable for focus